notebooks/create_synthetic_data.py

```
@@ -0,0 +1,22 @@
+spark = SparkSession.builder.getOrCreate()
+
+full_data = spark.read.table(f"{config.catalog}.{config.db_schema}.{config.use_case_name}")
+
+# COMMAND ----------
+existing_ids = full_data.select(config.primary_key).rdd.flatMap(lambda x: x).collect()
+
+# COMMAND ----------
+synthetic_df = generate_synthetic_data(config, full_data, config.primary_key)
+
+# COMMAND ----------
+synthetic_df.write.format("delta").mode("append").saveAsTable(f"{config.catalog}.{config.db_schema}.{config.use_case_name}")
```

Ruff

No newline at end of file

Add trailing newline

(W292)